tests/h/services/delete_user_test.py

```diff
@@ -73,14 +73,17 @@ def test_delete_user_removes_groups_if_no_collaborators(
         db_session.flush()
         assert sqlalchemy.inspect(group).was_deleted
 
-    def test_delete_user_fails_if_groups_have_collaborators(
+    def test_creator_is_unasigned_if_groups_have_collaborators(
         self, db_session, group_with_two_users, pyramid_request, svc
     ):
         pyramid_request.db = db_session
         (group, creator, member, creator_ann, member_ann) = group_with_two_users
 
-        with pytest.raises(UserDeleteError):
-            svc.delete(creator)
+        svc.delete(creator)
+
+        db_session.flush()
```

Comment:
Was there any particular reason for the `db_session.flush()` call here? The test passes without it.